extensions/BugModal/template/en/default/bug_modal/field.html.tmpl

```diff
@@ -219,7 +219,7 @@ END;
 
         [% CASE constants.FIELD_TYPE_FREETEXT %]
           [%# normal input field %]
-          <input name="[% name FILTER html %]" id="[% name FILTER html %]" value="[% value FILTER html %]" [% aria_labelledby_html FILTER none +%] [% " required" IF required %]>
+          <input name="[% name FILTER html %]" id="[% name FILTER html %]" value="[% value FILTER html %]" [% aria_labelledby_html FILTER none +%] [% "required" IF required %]>
```

Comment:
this change snuck back in. please add the space before the "required" string, so it is " required"